src/Engine/Objects/Actor.cpp

```diff
@@ -2699,7 +2689,7 @@ void Actor::UpdateActorAI() {
         }
 
         // If actor Paralyzed or Stoned: skip
-        if (pActor->pActorBuffs[ACTOR_BUFF_PARALYZED].Active() || pActor->pActorBuffs[ACTOR_BUFF_STONED].Active())
+        if (pActor->pActorBuffs[ACTOR_BUFF_PARALYZED].Active() || pActor->pActorBuffs[ACTOR_BUFF_STONED].Active() || pActor->uAIState == AIState::Stunned)
```

Comment:
This is a behavior change, is this vanilla?

If not, let's move this into a separate if block and add a comment --- what it does and why we're introducing this change.

Reply:
issue #774 added